content/general/limits.textile

```diff
@@ -298,57 +298,57 @@ Webhook batch concurrency is the number of webhook batches that can be processed
 
 h3(#firehose-messages). Firehose message rate
 
-The Firehose message rate is the maximum amount of Firehose messages that can be streamed per second.
+The Firehose message rate is the maximum rate at which Firehose messages can be streamed.
 
-If the Firehose message rate is exceeded, a "global rate limit":#global-rates on publishing to Firehose is applied.
+If the Firehose message rate is exceeded, a "global instantaneous rate limit":#global-rates on publishing to Firehose is applied.
 
 h3(#function-invocations). Function invocation rate
 
-The function invocation rate is the maximum number of functions that can be invoked per second. This includes AWS Lambda, Google Cloud and Azure functions that can be triggered by "events":https://ably.com/documentation/general/events.
+The function invocation rate is the maximum rate at which functions can be invoked. This includes AWS Lambda, Google Cloud and Azure functions that can be triggered by "events":https://ably.com/documentation/general/events.
```

Comment:
Would it be clearer to say "The function invocation rate _limit_ is the maximum rate at which functions can be invoked'? Without the "limit" in there, the function invocation rate is just the number of times you are calling the function. This might also be applicable elsewhere (e.g. Firehose message rate)

Reply:
Coincidentally I did have that originally but removed it for being too wordy. I've added it back in for all occurrences of rate limits though, as I didn't consider the inference you've pointed out, thanks.